internal/lockclient/simple_client.go

```diff
@@ -38,11 +38,13 @@ type SimpleClient struct {
 	// whether the process owning the lock has an active session
 	// or not, this guarantee has to be ensured by the client.
 	sessionAcquisitions map[id.ID][]lockservice.Descriptors
+
+	sessionDuration time.Duration
 }
 
 // NewSimpleClient returns a new SimpleClient of the given parameters.
 // This client works with or without the existance of a cache.
-func NewSimpleClient(config *lockservice.SimpleConfig, log zerolog.Logger, cache *cache.LRUCache) *SimpleClient {
+func NewSimpleClient(config *lockservice.SimpleConfig, log zerolog.Logger, cache *cache.LRUCache, sessionDuration time.Duration) *SimpleClient {
```

Comment:
This line is too long, try cascading them to lines

```suggestion
func NewSimpleClient(config *lockservice.SimpleConfig,
 log zerolog.Logger, 
 cache *cache.LRUCache,
  sessionDuration time.Duration
  ) *SimpleClient {
```

Something like this with lint ^